src/Uno.UWP/Graphics/Imaging/BitmapEncoder.skia.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using SkiaSharp;
+using Windows.Foundation;
+using System.IO;
+using System.Runtime.InteropServices;
+
+namespace Windows.Graphics.Imaging
+{
+	public partial class BitmapEncoder
```

Comment:
Do I have to implement a specific backend for each platform or can I use Skia for all of them?